src/hotspot/os_cpu/bsd_aarch64/os_bsd_aarch64.cpp

```diff
@@ -72,7 +72,9 @@
 # include <sys/wait.h>
 # include <pwd.h>
 # include <poll.h>
+#ifndef __OpenBSD__
 # include <ucontext.h>
```

Comment:
It probably makes more sense to just move this into the #ifdef __FreeBSD__ block immediately below here than wrap it in an #ifndef __OpenBSD__